namelayer-spigot/src/main/java/vg/civcraft/mc/namelayer/database/GroupManagerDao.java

```diff
@@ -356,7 +357,17 @@ public void run() {
 					if (g == null) {
 						createGroup(NameLayerPlugin.getSpecialAdminGroup(), null, null);
 					} else {
-						removeAllMembers(g.getName());
+						try (Connection connection = db.getConnection();PreparedStatement removeAllMembers = connection.prepareStatement("delete fm.* from faction_member fm "
+								+ "inner join faction_id fi on fi.group_id = fm.group_id where fi.group_name =?;"))  {
+							removeAllMembers.setString(1, g.getName());
+							removeAllMembers.execute();
+						} catch (SQLException e) {
+							e.printStackTrace();
+						}
+						
+						
+						cleanExecute("delete fm.* from faction_member fm inner join groupPlayerTypes gpt on gpt.type_id = fm.type_id "
+								+ "where gpt.group_id = ;");
```

Comment:
Ok. Mostly just trying to wrap my head around your goal; so I'm assuming you're just intending to remove all members from the special group, yeah?